sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -171,6 +171,32 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
     testDropTable(isDatasourceTable = false)
   }
 
+  test("DetermineTableStats should not cause any plan changes" +
```

Comment:
What would be right place to add this test? @viirya @maropu 
I will  move the table create/drop to `beforeAll` and `afterAll` when moving this test to the right file